R/reportFE.R

```diff
@@ -1344,10 +1368,13 @@ reportFE <- function(gdx, regionSubsetList = NULL,
   )
 
 
-  ### temporary (!) industry non-energy use reporting
+  #### Non-energy Use Reporting ----
+
+ ### temporary (!) industry non-energy use reporting
   # note: only for REMIND-EU SSP2
 
-  if ("DEU" %in% getRegions(vm_prodFe) & indu_mod == 'subsectors') {
+  if ("DEU" %in% getRegions(vm_prodFe) & indu_mod == 'subsectors' & is.null(vm_demFENonEnergySector)) {
+
```

Comment:
Yeah, I think our non-energy use reporting for ARIADNE can actually be removed once this is merged. Although you are right it is maybe safe to still keep it here and I will check out our new Germany scenarios with the feedstocks implementation afterwards and then delete this part, if everything looks good.